config_defaults/subtests/docker_cli/negativeusage.ini

```diff
@@ -1,5 +1,5 @@
 [docker_cli/negativeusage]
-subsubtests = op1, op2, op3, if1, ov1, iv1, iv3, iv4, iv5, iv6, iv7, ip1
+subsubtests = op1, op2, op3, if1, ov1, iv1, iv3, iv4, ip1
```

Comment:
Some of these are referenced in ``subtests.ini`` generated from template in ADEPT.  I'll fix that to allow CI to actually run this test w/o barfing.